src/cosmosdb-preview/azext_cosmosdb_preview/custom.py

```diff
@@ -1038,3 +1038,134 @@ def cli_table_retrieve_latest_backup_time(client,
+
+    if cassandra_table is None:
+        raise CLIError('Sink is required for dts import')
+
+    job_create_properties = {}
+    job_create_properties['source'] = blob_container
+    job_create_properties['destination'] = cassandra_table
+
+    if worker_count > 0:
+        job_create_properties['worker_count'] = worker_count
+
+    job_create_parameters = {}
+    job_create_parameters['properties'] = job_create_properties
+
+    if job_name is None:
+        job_name = _gen_guid()
+
+    return client.create(resource_group_name=resource_group_name,
+                         account_name=account_name,
+                         job_name=job_name,
+                         job_create_parameters=job_create_parameters)
+
+
+def cosmosdb_data_transfer_copy_job(client,
+                                    resource_group_name,
+                                    account_name,
+                                    source_cassandra_table=None,
+                                    dest_cassandra_table=None,
+                                    source_sql_container=None,
+                                    dest_sql_container=None,
+                                    worker_count=0,
+                                    job_name=None):
+    if source_cassandra_table is None and source_sql_container is None:
+        raise CLIError('source component ismissing')
+
+    if source_cassandra_table is not None and source_sql_container is not None:
+        raise CLIError('Invalid input: multiple source components')
+
+    if dest_cassandra_table is None and dest_sql_container is None:
+        raise CLIError('destination component is missing')
+
+    if dest_cassandra_table is not None and dest_sql_container is not None:
+        raise CLIError('Invalid input: multiple destination components')
+
+    job_create_properties = {}
+
+    if source_cassandra_table is not None:
+        job_create_properties['source'] = source_cassandra_table
+
+    if source_sql_container is not None:
+        job_create_properties['source'] = source_sql_container
+
+    if dest_cassandra_table is not None:
+        job_create_properties['destination'] = dest_cassandra_table
+
+    if dest_sql_container is not None:
+        job_create_properties['destination'] = dest_sql_container
+
+    if worker_count > 0:
+        job_create_properties['worker_count'] = worker_count
+
+    job_create_parameters = {}
+    job_create_parameters['properties'] = job_create_properties
+
+    if job_name is None:
+        job_name = _gen_guid()
+
+    return client.create(resource_group_name=resource_group_name,
+                         account_name=account_name,
+                         job_name=job_name,
+                         job_create_parameters=job_create_parameters)
+
+
+def cosmosdb_dts_list(client,
+                      resource_group_name,
+                      account_name):
+    return client.list_by_database_account(resource_group_name=resource_group_name,
```

Comment:
Since there is no additional operations, you can remove this and change the custom_command to command